packages/xod-cli/src/commands/publish.js

```diff
@@ -0,0 +1,178 @@
+import { exit } from 'process';
+import { pick } from 'ramda';
+import * as xodFs from 'xod-fs';
+import {
+  getProjectName,
+  getProjectDescription,
+  getProjectVersion,
+} from 'xod-project';
+import { cli } from 'cli-ux';
+import BaseCommand from '../baseCommand';
+import commonArgs from '../args';
+import * as myFlags from '../flags';
+import * as msg from '../messages';
+import {
+  getAccessToken,
+  getLib,
+  getUser,
+  postUserLib,
+  putUserLib,
+} from '../apis';
+
+const packLibVersion = async (workspaceDir, projectDir) => {
+  const projectWithoutLibs = await xodFs.loadProjectWithoutLibs(
+    await xodFs.findClosestProjectDir(projectDir),
+    workspaceDir || (await xodFs.findClosestWorkspaceDir(projectDir))
```

Comment:
Looks like the second argument is not needed here:
https://github.com/xodio/xod/blob/master/packages/xod-fs/src/load.js#L124

Reply:
:open_mouth: Ooops, I port this from old version https://github.com/xodio/xod/blob/master/packages/xod-cli/src/xodc-publish.js#L12